README.md

```diff
@@ -2,7 +2,7 @@
 
 [Agile Toolkit](https://atk4.org/) is a Low Code framework written in PHP. Agile UI implement server side rendering engine and over 50 UI generic components for interacting with your Data Model.
 
-Agile Data is a framework for defining your "business layer" which is separate from your "presentation layer" and "persistence". Together with [Agile UI](https://github.com/atk4/ui) you can deliver user interface "out of the box" or with [Agile UI](https://github.com/atk4/api) - general-purpose API endpoints.
+Agile Data is a framework for defining your "business layer" which is separate from your "presentation layer" and "persistence". Together with [Agile UI](https://github.com/atk4/ui) you can deliver user interface "out of the box" or with [Atk4\Api](https://github.com/atk4/api) - general-purpose API endpoints.
```

Comment:
I completely agree to use one name. In most of the places, we use `atk4/xxx` (in LC, and forward slash). What about converting every "Agile Xxx" to `atk4/xxx`?

Reply:
Hi, I replaced it with atk4/api. I agree that we should have a clear naming. This PR was just a quick fix - these outdated links will scare away any new developer checking the repo. When I have more time, I can do more work on the readme like renaming, updated code samples and also some restructuring.